app/src/main/java/ar/com/wolox/android/training/ui/root/RootPresenter.kt

```diff
@@ -0,0 +1,17 @@
+package ar.com.wolox.android.training.ui.root
+
+import android.content.SharedPreferences
+import ar.com.wolox.wolmo.core.presenter.BasePresenter
+import javax.inject.Inject
+
+class RootPresenter @Inject constructor(private val sharedPreferences: SharedPreferences) : BasePresenter<IRootView>() {
+
+    fun loadUserPreferences() {
+        val vUserEmail = sharedPreferences.getString("UserEmail", "")
```

Comment:
You need to do the same here with the companion object :)

Reply:
Solved in next PR update.